### PR TITLE
New: Suspicious Activity model

### PR DESCRIPTION
**Pull request by ericrpowers**
#### Description
This covers the suspicious activity model that we went to cover generic suspicious events from a flow report, DNS report, etc. When something will be suspicious, we will fill the information like the `sourceType`, `sourceID` and `sourceNamespace` we can retrieve all the related info.

> Fixes: https://redlock.atlassian.net/browse/CNS-5040

### CODE REVIEW

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "suspicious-activity",
    "component": "apomux",
    "pr-id": "149",
    "commit-sha": "a5710a073b377b1782c26ad83acb481ab7c44220",
    "pipeline": "master"
  },
  {
    "project": "suspicious-activity",
    "component": "backend",
    "pr-id": "1762",
    "commit-sha": "cc0e81ad83970c512adc950a084075b319d07295",
    "pipeline": "master"
  },
  {
    "project": "suspicious-activity",
    "component": "gaia",
    "pr-id": "813",
    "commit-sha": "af169da2335d9608f223c64c5a389a68b70bf97d",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1762",
    "commit-sha": "cc0e81ad83970c512adc950a084075b319d07295",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "813",
    "commit-sha": "af169da2335d9608f223c64c5a389a68b70bf97d",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "813",
    "commit-sha": "af169da2335d9608f223c64c5a389a68b70bf97d",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1762",
    "commit-sha": "363216fd49c66dd2a894d3ec6c474b68e7815f6e",
    "pipeline": "master"
  }
]
```

File: specs/unknownipaddress.spec [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  - '@identifiable-stored'
+  - '@namespaced'
+  - '@timeable'
+  - '@zoned'
+
+# Indexes
+indexes:
+- - namespace
+  - address
+- - createTime
```
Comment (primalmotion) [inline specs/unknownipaddress.spec:20]: this should probably be namespaced

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1762",
    "commit-sha": "b34cfa9258e1182b1634bc63e73821ab98127c69",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "813",
    "commit-sha": "0f0cfc9daf61043001395b44ae622ac3a452ab29",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1762",
    "commit-sha": "fd205a38daade3fc768529cb14a04645bb1e03ed",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "813",
    "commit-sha": "117a8451856d483fd356367ed979cc30a3b084c2",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "voila",
    "pr-id": "194",
    "commit-sha": "9a046dc16e3b4a31ee0c3dd3a317a4702f32c140",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1762",
    "commit-sha": "fd205a38daade3fc768529cb14a04645bb1e03ed",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "813",
    "commit-sha": "3cbce34ff447d7d1c20c5a59564031f23b59b693",
    "pipeline": "master"
  }
]
```

**Review by t00f — APPROVED** [review]

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1762",
    "commit-sha": "495e6330f7f4ec0435825ad64f6a46e78786abad",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "813",
    "commit-sha": "3cbce34ff447d7d1c20c5a59564031f23b59b693",
    "pipeline": "master"
  }
]
```

**Pull request merged**